testkit/integration.py

```diff
@@ -8,7 +8,8 @@ def run(args):
 
 
 def test_driver():
-    run(["gulp", "test-browser"])
+    if os.environ.get("TEST_DRIVER_SKIP_BROWSER", False):
```

Comment:
This is equivalent to `if "TEST_DRIVER_SKIP_BROWSER" in os.environ`. Did you maybe mean something along the lines of 
```suggestion
    if (os.environ.get("TEST_DRIVER_SKIP_BROWSER", "false").lower()
            in ("y", "yes", "t", "true", "1")):
```

Reply:
Thanks, It does make sense.